### PR TITLE
Adds Windows Server 2016 images to LabMedia

### PR DESCRIPTION
**Pull request by csandfeld**

### CODE REVIEW

File: Config/Media.json [hunk 1]
```diff
@@ -1,4 +1,88 @@
...
+[
+{
+    "Id":  "2016_x64_Standard_EN_Eval",
+    "Filename":  "2016_x64_EN_Eval.iso",
+    "Description":  "Windows Server 2016 Standard 64bit English",
+    "Architecture":  "x64",
+    "ImageName":  "Windows Server 2016 SERVERSTANDARD",
+    "MediaType":  "ISO",
+    "OperatingSystem":  "Windows",
+    "Uri":  "http://care.dlservice.microsoft.com/dl/download/1/6/F/16FA20E6-4662-482A-920B-1A45CF5AAE3C/14393.0.160715-1616.RS1_RELEASE_SERVER_EVAL_X64FRE_EN-US.ISO",
```
Comment (iainbrighton) [inline Config/Media.json]: Can you change the URL to http://download.microsoft.com/download/1/6/F/16FA20E6-4662-482A-920B-1A45CF5AAE3C/14393.0.160715-1616.RS1_RELEASE_SERVER_EVAL_X64FRE_EN-US.ISO

File: Config/Media.json [hunk 2]
```diff
@@ -1,4 +1,88 @@
...
+},
+{
+    "Id":  "2016_x64_Standard_Core_EN_Eval",
+    "Filename":  "2016_x64_EN_Eval.iso",
+    "Description":  "Windows Server 2016 Standard Core 64bit English",
+    "Architecture":  "x64",
+    "ImageName":  "Windows Server 2016 SERVERSTANDARDCORE",
+    "MediaType":  "ISO",
+    "OperatingSystem":  "Windows",
+    "Uri":  "http://care.dlservice.microsoft.com/dl/download/1/6/F/16FA20E6-4662-482A-920B-1A45CF5AAE3C/14393.0.160715-1616.RS1_RELEASE_SERVER_EVAL_X64FRE_EN-US.ISO",
```
Comment (iainbrighton) [inline Config/Media.json]: Same URL

File: Config/Media.json [hunk 3]
```diff
@@ -1,4 +1,88 @@
...
+},
+{
+    "Id":  "2016_x64_Datacenter_EN_Eval",
+    "Filename":  "2016_x64_EN_Eval.iso",
+    "Description":  "Windows Server 2016 Datacenter 64bit English",
+    "Architecture":  "x64",
+    "ImageName":  "Windows Server 2016 SERVERDATACENTER",
+    "MediaType":  "ISO",
+    "OperatingSystem":  "Windows",
+    "Uri":  "http://care.dlservice.microsoft.com/dl/download/1/6/F/16FA20E6-4662-482A-920B-1A45CF5AAE3C/14393.0.160715-1616.RS1_RELEASE_SERVER_EVAL_X64FRE_EN-US.ISO",
```
Comment (iainbrighton) [inline Config/Media.json]: Same URL

File: Config/Media.json [hunk 4]
```diff
@@ -1,4 +1,88 @@
...
+},
+{
+    "Id":  "2016_x64_Datacenter_Core_EN_Eval",
+    "Filename":  "2016_x64_EN_Eval.iso",
+    "Description":  "Windows Server 2016 Datacenter Core 64bit English",
+    "Architecture":  "x64",
+    "ImageName":  "Windows Server 2016 SERVERDATACENTERCORE",
+    "MediaType":  "ISO",
+    "OperatingSystem":  "Windows",
+    "Uri":  "http://care.dlservice.microsoft.com/dl/download/1/6/F/16FA20E6-4662-482A-920B-1A45CF5AAE3C/14393.0.160715-1616.RS1_RELEASE_SERVER_EVAL_X64FRE_EN-US.ISO",
```
Comment (iainbrighton) [inline Config/Media.json]: Same URL

File: Config/Media.json [hunk 5]
```diff
@@ -1,4 +1,88 @@
-[{
+[
+{
+    "Id":  "2016_x64_Standard_EN_Eval",
+    "Filename":  "2016_x64_EN_Eval.iso",
+    "Description":  "Windows Server 2016 Standard 64bit English",
```
Comment (iainbrighton) [inline Config/Media.json]: Windows Server 2016 Standard 64bit English Evaluation

File: Config/Media.json [hunk 6]
```diff
@@ -1,4 +1,88 @@
...
+    "Checksum":  "18A4F00A675B0338F3C7C93C4F131BEB",
+    "CustomData":  {
+        "WindowsOptionalFeature":  ["NetFx3"]
+    },
+    "Hotfixes":  []
+},
+{
+    "Id":  "2016_x64_Standard_Core_EN_Eval",
+    "Filename":  "2016_x64_EN_Eval.iso",
+    "Description":  "Windows Server 2016 Standard Core 64bit English",
```
Comment (iainbrighton) [inline Config/Media.json]: Windows Server 2016 Standard Core 64bit English Evaluation

File: Config/Media.json [hunk 7]
```diff
@@ -1,4 +1,88 @@
...
+    "Checksum":  "18A4F00A675B0338F3C7C93C4F131BEB",
+    "CustomData":  {
+        "WindowsOptionalFeature":  ["NetFx3"]
+    },
+    "Hotfixes":  []
+},
+{
+    "Id":  "2016_x64_Datacenter_Core_EN_Eval",
+    "Filename":  "2016_x64_EN_Eval.iso",
+    "Description":  "Windows Server 2016 Datacenter Core 64bit English",
```
Comment (iainbrighton) [inline Config/Media.json]: Windows Server 2016 Datacenter Core 64bit English Evaluation

File: Config/Media.json [hunk 8]
```diff
@@ -1,4 +1,88 @@
...
+    "OperatingSystem":  "Windows",
+    "Uri":  "http://care.dlservice.microsoft.com/dl/download/1/6/F/16FA20E6-4662-482A-920B-1A45CF5AAE3C/14393.0.160715-1616.RS1_RELEASE_SERVER_EVAL_X64FRE_EN-US.ISO",
+    "Checksum":  "18A4F00A675B0338F3C7C93C4F131BEB",
+    "CustomData":  {
+        "WindowsOptionalFeature":  ["NetFx3"]
+    },
+    "Hotfixes":  []
+},
+{
+    "Id":  "2016_x64_NANO_EN_Eval",
```
Comment (iainbrighton) [inline Config/Media.json]: There are actually two Nano images:
```powershell
C:\..\Dropbox\PowerShell\> Get-WindowsImage -ImagePath D:\Nano\NanoServer.wim

ImageIndex       : 1
ImageName        : Windows Server 2016 SERVERSTANDARDNANO
ImageDescription :
ImageSize        : 800,233,355 bytes

ImageIndex       : 2
ImageName        : Windows Server 2016 SERVERDATACENTERNANO
ImageDescription :
ImageSize        : 811,183,747 bytes
```
Rename this one to `2016_x64_Standard_Nano_EN_Eval`

File: Config/Media.json [hunk 9]
```diff
@@ -1,4 +1,88 @@
...
+    "Filename":  "2016_x64_EN_Eval.iso",
+    "Description":  "Windows Server 2016 Nano 64bit English",
```
Comment (iainbrighton) [inline Config/Media.json]: Rename to `Windows Server 2016 Standard Nano 64bit English Evaluation`

File: Config/Media.json [hunk 10]
```diff
@@ -1,4 +1,88 @@
...
+    "Architecture":  "x64",
+    "ImageName":  "1",
```
Comment (iainbrighton) [inline Config/Media.json]: And update the image name to `Windows Server 2016 SERVERSTANDARDNANO`

File: Config/Media.json [hunk 11]
```diff
@@ -1,4 +1,88 @@
...
+    "MediaType":  "ISO",
+    "OperatingSystem":  "Windows",
+    "Uri":  "http://care.dlservice.microsoft.com/dl/download/1/6/F/16FA20E6-4662-482A-920B-1A45CF5AAE3C/14393.0.160715-1616.RS1_RELEASE_SERVER_EVAL_X64FRE_EN-US.ISO",
```
Comment (iainbrighton) [inline Config/Media.json]: Same URL

File: Config/Media.json [hunk 12]
```diff
@@ -1,4 +1,88 @@
...
+        "Package":  [
+            "Microsoft-NanoServer-Guest-Package",
+            "Microsoft-NanoServer-DSC-Package",
+            "Microsoft-NanoServer-Compute-Package",
+            "Microsoft-NanoServer-Containers-Package"
+        ]
+    },
+    "Hotfixes":  []
+},
+{
```
Comment (iainbrighton) [inline Config/Media.json]: Can you also add a `Windows Server 2016 Datacenter Nano 64bit English Evaluation` media entry.

File: Config/Media.json [hunk 13]
```diff
@@ -1,4 +1,88 @@
...
 	"Id": "2016TP5_x64_Standard_EN",
```
Comment (iainbrighton) [inline Config/Media.json]: Now we're GA, can you remove all the Technical Preview media?

File: Config/Media.json (hunk 1 above)
Comment (csandfeld) [inline Config/Media.json]: Sure. Noticed the care.dlservice... url used for the TP5 media, and found it to be faster when I downloaded the RTM. Hence why I used that.

Implementing this and the other changes, and will submit once I downloaded the ISO from your URL and verified the CheckSum.

Comment (iainbrighton) [inline Config/Media.json]: I've never considered (or tested) the download speed from download.microsoft.com compared to care.dlservice.microsoft.com. @ryanspletzer do you have any thoughts/comments?

Comment (csandfeld) [inline Config/Media.json]: Currently downloading the RTM from download.microsoft.com, and up to 44% by now (after 25 min or so)

Comment (csandfeld) [inline Config/Media.json]: Did some testing, and found that today download speed was more or less equal  between download.microsoft-com and care.dlservice.microsoft.com (with download.microsoft.com actually finishing a couple of minutes faster). So probably no need to persue that any further.

**Pull request merged**